managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/EntryImpl.java

```diff
@@ -66,6 +67,17 @@ public static EntryImpl create(long ledgerId, long entryId, byte[] data) {
         return entry;
     }
 
+    public static EntryImpl create(long ledgerId, long entryId, boolean skipped) {
```

Comment:
what about "createSkippedEntry" ?
otherwise people may want to use this factory method for other purposes

Reply:
OK

Reply:
Fixed! PTAL,thanks! @eolivelli